toonz/sources/toonz/xshcolumnviewer.cpp

```diff
@@ -1474,12 +1411,13 @@ void ColumnArea::openTransparencyPopup() {
 
 //----------------------------------------------------------------
 
-void ColumnArea::startTransparencyPopupTimer(QMouseEvent *e) {
+void ColumnArea::startTransparencyPopupTimer(QMouseEvent *e) {  // AREA
   if (!m_columnTransparencyPopup)
     m_columnTransparencyPopup = new ColumnTransparencyPopup(
         this);  // Qt::ToolTip|Qt::MSWindowsFixedSizeDialogHint);//Qt::MSWindowsFixedSizeDialogHint|Qt::Tool);
 
-  int x = e->pos().x() - m_tabBox.left() - m_viewer->columnToX(m_col);
+  int x = e->pos().x() - m_tabBox.left() -
+          m_viewer->columnToLayerAxis(m_col);  // what is happening here?
```

Comment:
`ColumnTransparencyPopup` appears when clicking & holding the camstand button (orange button) in the column. It is for controlling opacity of the column and applying a color filter to the raster level column.
For now the popup seems to appears incorrect position in the Timeline mode.